firmware_hosting/github-access.tf

```diff
@@ -0,0 +1,24 @@
+data "cloudflare_api_token_permission_groups" "all" {}
+
+resource "cloudflare_api_token" "github-token" {
+  name = "github-firmware-upload-token"
```

Comment:
This should probably hint to that it is being created in TF.

Reply:
How would we best do this?

Reply:
I would prefix it with `terraform-` so the name would be `terraform-github-firmware-upload-token`